docs/guides/code_push_quickstart.md

```diff
@@ -16,18 +16,13 @@ Before you can create a Shorebird app, you will need to sign up for Shorebird.
 
 ### Create an account
 
-To register as a Shorebird user, run:
+To create an account, head over to the [Shorebird Console](https://console.shorebird.dev) and authenticate with your Google account.
 
-```sh
-shorebird account create
-```
-
-This will generate a Google sign in link. Follow the link and sign in with your
-Google account.
+Once you have logged into the console, follow the instructions to install the Shorebird CLI on your machine.
 
 ## Create the app
 
-Once you have registered, you're ready to use Shorebird!
+Once you have registered and installed the CLI, you're ready to use code push!
```

Comment:
Curious to hear @eseidel's thoughts on this, but I think "Shorebird" might be preferable here. My (potentially wrong) instinct is that Shorebird and code push will be synonymous in our customers' minds until we have a second product.

Reply:
Happy to revert my thinking was this is a code push quick start guide so at this point in the guide they've installed Shorebird and are ready to use code push (the product). Will wait to hear @eseidel's thoughts and happy to adjust accordingly 👍

Reply:
Landing as is based on Discord discussion but happy to adjust in the future 👍